platforms/intellij/compose/playground/src/jvmMain/kotlin/foundry/intellij/compose/playground/MarkdownPlayground.kt

```diff
@@ -31,11 +33,19 @@ import org.jetbrains.jewel.intui.standalone.theme.IntUiTheme
 import org.jetbrains.jewel.ui.component.DefaultButton
 import org.jetbrains.jewel.ui.component.Text
 
-fun main() = singleWindowApplication {
+fun main() {
+  singleWindowApplication { MarkdownPlayground() }
+}
+
+@Composable
+fun MarkdownPlayground(modifier: Modifier = Modifier) {
   var isDark by remember { mutableStateOf(false) }
   IntUiTheme(isDark) {
-    Column(Modifier.background(JewelTheme.globalColors.panelBackground)) {
-      DefaultButton(modifier = Modifier.padding(16.dp), onClick = { isDark = !isDark }) {
+    Column(modifier.background(JewelTheme.globalColors.panelBackground)) {
+      DefaultButton(
+        modifier = Modifier.padding(16.dp).testTag("dark-mode-toggle"),
```

Comment:
nit: pull the test tag out to a shared constant so you can reuse that reference in test code too. That way if you update its value both get the update. Usually it's something like a `object TestTags { ... }`

Reply:
I added this object and referred to it here and in the test: 992b80dd7906e3e8b7bf586d7afed8d3bf7cfe69